upgrade_test.py

```diff
@@ -789,15 +809,23 @@ def test_kubernetes_scylla_upgrade(self):
         self.db_cluster.upgrade_scylla_cluster(target_upgrade_version)
         self.log.info('Step6 - Wait till cluster got upgraded')
         self.wait_till_scylla_is_upgraded_on_all_nodes(target_upgrade_version)
+
         self.log.info('Step7 - Upgrade sstables')
-        upgradesstables = self.db_cluster.run_func_parallel(func=self.upgradesstables_if_command_available)
-        # only check sstable format version if all nodes had 'nodetool upgradesstables' available
-        if all(upgradesstables):
-            self.log.info("Waiting until jmx is up across the board")
-            self.wait_till_jmx_on_all_nodes()
-            self.log.info('Upgrading sstables if new version is available')
-            tables_upgraded = self.db_cluster.run_func_parallel(func=self.wait_for_sstable_upgrade)
-            assert all(tables_upgraded), f"Failed to upgrade the sstable format {tables_upgraded}"
+        if self.params.get('upgrade_sstables'):
```

Comment:
Why was this change made?
AFAIK we removed the dependency of params.get so we won't need to use it.

Also, I don't see this param in the test_yaml of the rolling upgrades, so are we counting on the default?
What is the default?

Reply:
The default is "false".
Usage of "upgrade_sstables" was added because it is used in other upgrade tests in addition to the fact that we have config option with possibility to disable it if needed.